### ISSUE TITLE
Settings UI : some settings difficult or impossible to change

### ISSUE THREAD

**Issue by RichCaloggero** [microsoft/vscode#121767]
This seems rather flakey, so it may have something to do with scrolling or other visual state that I can't discern.

Not sure if this requires a screen reader to be running in order to show the issue; try with NVDA if possible.  Do not use mouse for any interaction...

- open user settings;
- using keyboard, tab into tree and press the end key (should land on "extensions")
- tab to list and press end key (I focus on "typescript workspace symbol scope")
- press tab or enter to focus the options
- press enter to modify (for me nothing happens when I do this)
- on windows, press alt+downArrow to open list (for me, nothing happens)
- focus on settings list again and press home key (should focus "commonly used")
- press downArrow (I hear "files autosave")
- press enter or tab, press enter again or alt+downArrow on windows and I can now see choices: off, after delay, on, on window change)

Sometimes I can convince NVDA to show choices by using NVDA's mouse keys, but often doesn't work.

**Comment by roblourens** [comment]
> on windows, press alt+downArrow to open list (for me, nothing happens)

For me, when I do this it expands the list, reads that the list is expanded, and reads the details of the selected option. It sounds like it works for you on the autosave setting, which is the same type of setting. I played around with this for a bit and it worked every time, so I'm not sure what's happening on your machine, sorry. Any idea how else to investigate this @isidorn?
